jwst/assign_wcs/nirspec.py

```diff
@@ -444,7 +453,9 @@ def get_open_msa_slits(msa_file, msa_metadata_id, slit_y_range=[-.55, .55]):
         ('background', 'S1'),
         ('shutter_state', 'S6'),
         ('estimated_source_in_shutter_x', '>f4'),
-        ('estimated_source_in_shutter_y', '>f4')])
+        ('estimated_source_in_shutter_y', '>f4'),
+        ('dither_point_index', '>i2'),
+        ('primary_source', 'S1')
 
     For example, something like:
         (12, 2, 4, 251, 22, 1, 'Y', 'OPEN', nan, nan),
```

Comment:
Should probably expand the example here to include the 2 additional arguments on the end